src/Sulu/Bundle/SearchBundle/Tests/Unit/Controller/WebsiteSearchControllerTest.php

```diff
@@ -93,7 +94,54 @@ public function testQueryAction()
             $searchQueryBuilder->reveal()
         );
         $searchQueryBuilder->locale('en')->willReturn($searchQueryBuilder->reveal());
-        $searchQueryBuilder->index('page_sulu_published')->willReturn($searchQueryBuilder->reveal());
+        $searchQueryBuilder->indexes(['page_sulu_published'])->willReturn($searchQueryBuilder->reveal());
+        $searchQueryBuilder->execute()->willReturn([]);
+
+        $this->parameterResolver->resolve(
+            ['query' => 'Test', 'hits' => []],
+            $this->requestAnalyzer->reveal()
+        )->willReturn(['query' => 'Test', 'hits' => []]);
+
+        $this->twigLoader->exists(Argument::any())->willReturn(true);
+
+        $this->twig->render(
+            'search.html.twig',
+            ['query' => 'Test', 'hits' => []]
+        )->willReturn(new Response());
+
+        $this->assertInstanceOf(Response::class, $this->websiteSearchController->queryAction($request));
+    }
+
+    public function testQueryActionWithDifferentIndexes()
+    {
+        $this->websiteSearchController = new WebsiteSearchController(
+            $this->searchManager->reveal(),
+            $this->requestAnalyzer->reveal(),
+            $this->parameterResolver->reveal(),
+            $this->twig->reveal(),
+            ['examples_published']
```

Comment:
Would make this a test with multiple indices and a `#webspace#' placeholder, you can't really tell until now if that is working by the tests, right?

Reply:
fixed